coderedcms/tests/testapp/models.py

```diff
@@ -0,0 +1,141 @@
+from modelcluster.fields import ParentalKey
```

Comment:
Please add docstring to top of this file indicating that we should create a concrete version of every one of our abstract models here to test them.